.reuse/dep5

```diff
@@ -36,3 +36,7 @@ License: Apache-2.0
 Files: bittide-instances/tests/reports/*
 Copyright: 2022 Google LLC
 License: Apache-2.0
+
+Files: verilog-ethernet
```

Comment:
```suggestion
Files: verilog-ethernet/*
```

? Not sure, but this is what the other sections do.

Reply:
Since all the included files already have licensing, I suspect that we are in the clear copyright wise. However, I had to add a MIT license text that _should_ include year and name of the copyright holder and I'm not sure if the way I added it now is correct. Can you take a look?

Reply:
Yeah I think it's fine